openlibrary/plugins/upstream/code.py

```diff
@@ -88,11 +88,15 @@ def GET(self):
 
 
 class merge_work(delegate.page):
-    path = "(/works/OL\d+W)/merge"
-    def GET(self, key):
-        return "This looks like a good place for a merge UI!"
+    path = "/works/merge"
```

Comment:
Is it right that the path is reversed between lines 91 and 97?

Reply:
Yeah, changed the format to match /authors/merge